docs/cookbook/setting_up_multi_language_siteaccesses.md

```diff
@@ -0,0 +1,142 @@
+# Setting up multi-language SiteAccesses and corresponding translations
+
+## Description
+
+Combining translated content with multiple SiteAccesses successfully can be a little tricky to get your head around when you are new to eZ Platform.
+
+To successfully achieve this for the most typical setups you will need to do all 3 following items:
+
+1. Register your translations in the database via the eZ Platform admin.
+1. Create at least two SiteAccesses to deliver the right translated content at the right time in `ezplatform.yml`.
+1. Set the correct permissions for the anonymous user to read each SiteAccess.
+
+Without these 3 steps, your SiteAccess configuration will either not work or you will be left with duplicate content from an SEO perspective.
+
+## 1. Create your new translation
+
+Here is the full guide on [Internationalization](../guide/internationalization.md).
+
+1. Log into your eZ Platform admin.
+1. Navigate to the Admin Panel -> Languages tab.
+1. Click "Create a new language" and follow the on-screen prompts to register your new language (e.g. `fre-FR`).
+1. After saving the new language, refresh the eZ Platform admin assets: `php app/console assetic:dump`.
+
+That's it! Reload the admin and you can now start translating content.
+
+## 2. Creating the new SiteAccesses
+
+Here is the full guide on [SiteAccess](../guide/siteaccess.md).
+
+The most typical setup for a site with translated content is to map the base of the domain to one language and use the first segment of the URI to match to another. For example:
+- www.mysite.com to match to English
+- www.mysite.com/fr to match to French
+
+To achieve this you will need to create at least two new SiteAccesses in your `ezplatform.yml` file.
+
+The first bit of this working example lists the new SiteAccesses `en` and `fr` and adds them both to a common group `site_group` (this group will be used for shared settings such as API keys, cache locations, etc.).
+
+``` yaml
+    siteaccess:
+        default_siteaccess: site
+        list:
+            - site
```

Comment:
A bit late, but isn't "site" a bit confusing here ? It is actually the `en` siteaccess from the example above, as we access english content by going to `mysite.com/` (and not `en`), unlike what is described below.